code/game/objects/structures/shower.dm

```diff
@@ -165,8 +165,8 @@
 		if(ishuman(M))
 			var/mob/living/carbon/human/H = M
 			if(check_clothes(L))
```

Comment:
in a future PR we should add onfire checks here so you can offset the fire moodlet with the shower moodlet (the shower will obviously feel great since it made your flesh stop burning).